src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, value) {
+      this.result = operation(this.result, value);
```

Comment:
Use an assignment operator combined with the arithmetic operation to update the `result` property. For example, if the operation is addition, use `this.result += value;` instead of `this.result = this.result + value;`. This will make the code cleaner and adhere to the checklist item number 2.